components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -48,9 +49,41 @@ export default function MeasureHighlightingPanel({ patientId }: MeasureHighlight
     return { parsedHTML, defIds };
   }, [detailedResultLookup, patientId]);
 
+  // handle search
+  const [searchValue, setSearchValue] = useState('');
+
   return (
     <>
-      <PopulationComparisonTable patientId={patientId} defIds={defIds} />
+      <Accordion chevronPosition="left" defaultValue="table">
```

Comment:
I feel like it make more sense to use `Collapse` instead of Accordion since usually Accordions are for things with multiple items that you want to toggle between. However, that's more work, so this is fine

Reply:
Yeah- I ended up going this direction because basically all the good examples I could find for collapsible components were styled in this way, and I wanted to rely on user familiarity with existing component styles. I think this makes it look usable in the right way and also lets us easily work with collapsing individual pieces of the table in the future if we want to do that.